doc/Known_issues/Service_restart_workarounds.md

```diff
@@ -0,0 +1,48 @@
+The purpose of this document is to describe problems with workarounds for restarting Contrail components on Windows.
```

Comment:
Please test this document with mkdocs, because I'm pretty sure a title needs to be added in the first line.

Reply:
The title will be the name of the file with underscores replaced by whitespaces.